src/cljs/rems/application.cljs

```diff
@@ -448,13 +448,11 @@
   [{:keys [element-id attributes application group? can-remove? accepted-licenses?]}]
   (let [application-id (:application/id application)
         user-id (or (:eppn attributes) (:userid attributes))
-        sanitized-user-id (-> (or user-id (:email attributes) "") ;; use email for invited members
-                              str/lower-case
-                              (str/replace #"[^a-z]" ""))
+        user-id-or-email (or user-id (:email attributes) "") ; use email for invited members
```

Comment:
> ID and NAME tokens must begin with a letter ([A-Za-z]) and may be followed by any number of letters, digits ([0-9]), hyphens ("-"), underscores ("_"), colons (":"), and periods (".").

At least these are safe, since that's HTML4's definition. Newer HTML doesn't have restrictions, but I don't know about browser compatibility.